docs/ampform/LambdaKpi0.ipynb

```diff
@@ -99,101 +95,138 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "mystnb": {
-     "code_prompt_show": "Lambda"
-    },
-    "tags": [
-     "hide-cell"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "particle_db[\"Lambda\"]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "mystnb": {
-     "code_prompt_show": "K+"
-    },
-    "tags": [
-     "hide-cell"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "particle_db[\"K+\"]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "mystnb": {
-     "code_prompt_show": "pi0"
-    },
-    "tags": [
-     "hide-cell"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "particle_db[\"pi0\"]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "mystnb": {
-     "code_prompt_show": "gamma"
-    },
-    "tags": [
-     "hide-cell"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "particle_db[\"gamma\"]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "mystnb": {
-     "code_prompt_show": "p"
+    "jupyter": {
+     "source_hidden": true
     },
     "tags": [
-     "hide-cell"
+     "hide-input"
     ]
    },
    "outputs": [],
    "source": [
-    "particle_db[\"p\"]"
+    "particles = [\"Lambda\", \"K+\", \"pi0\", \"gamma\", \"p\"]\n",
+    "latex_particle_names = {\n",
+    "    \"Lambda\": r\"\\Lambda\",\n",
+    "    \"K+\": r\"K^+\",\n",
+    "    \"pi0\": r\"\\pi^0\",\n",
+    "    \"gamma\": r\"\\gamma\",\n",
+    "    \"p\": \"p\",\n",
+    "}\n",
+    "\n",
+    "\n",
+    "def format_parity(parity: int | None) -> str:\n",
+    "    if parity is None:\n",
+    "        return \"\"\n",
+    "    if parity == -1:\n",
+    "        return \"-\"\n",
+    "    if parity == 1:\n",
+    "        return \"+\"\n",
+    "    return str(parity)\n",
```

Comment:
I would `raise NotImplementedError` to avoid feeding this function the wrong input